cdap-watchdog/src/main/java/co/cask/cdap/logging/plugins/LocationManager.java

```diff
@@ -195,8 +194,12 @@ Location getLogLocation(LocationIdentifier locationIdentifier) throws IOExceptio
   }
 
   @VisibleForTesting
-  Map<LocationIdentifier, LocationOutputStream> getInvalidOutputStreams() {
-    return invalidOutputStreams;
+  LocationOutputStream getInvalidOutputStream() {
```

Comment:
also this class is not used in tests, can remove the `visiblefortesting` annotation.